configuration/configs/production.json

```diff
@@ -538,6 +714,12 @@
       "connections": ["moonbeam", "milkomedaC1"],
       "manualProcessing": true,
       "connextEnabled": true
+    },
+    "evmos": {
+      "displayName": "Evmos",
+      "nativeTokenSymbol": "EVMOS",
+      "connections": ["ethereum"],
+      "connextEnabled": true
```

Comment:
I think this should be false initially.  Don't know what the state of Connext deploys are on evmos